types/priv_validator.go

```diff
@@ -105,13 +97,14 @@ func LoadPrivValidator(filePath string) *PrivValidator {
 	if err != nil {
 		Exit(err.Error())
 	}
-	privVal := wire.ReadJSON(&PrivValidator{}, privValJSONBytes, &err).(*PrivValidator)
+	privVal := PrivValidator{}
```

Comment:
I don't have a preference, but since the only use of `privVal` here is the pointer, I'd rather use `new` than have to include `&` everywhere later